apps/example-todo-app/tests/api/todo/add.test.ts

```diff
@@ -1,14 +1,15 @@
 import test from "ava"
-import { TODO_ID } from "tests/fixtures"
 import getTestServer from "tests/fixtures/get-test-server"
-import { v4 as uuidv4 } from "uuid"
 
 test("POST /todo/add", async (t) => {
   const { axios } = await getTestServer(t)
 
   const noAuthRes = await axios.post("/todo/add").catch((err) => err)
   t.is(noAuthRes.status, 401, "no auth")
 
+  const hasErrorStack = Boolean(noAuthRes.response.error.stack);
```

Comment:
@seveibar do you think this is enough?

Reply:
yea that's pretty good!

Reply:
let's check for an error code for an explicit exception

Reply:
so if i throw `BadRequestException({ type: "my_error_code", message: "hello world" })`

we should see in our response
`{ error: { error_code, message } }`

Reply:
@seveibar Actually, we can see the error object even without the error handling; the only new data to the error object is the stack object